anvio/__init__.py

```diff
@@ -2177,6 +2177,13 @@ def TABULATE(table, header, numalign="right", max_width=0):
                      "This way you can run HMM profiles that are not included in anvi'o. See the online "
                      "to find out about the specifics of this directory structure ."}
                 ),
+    'hmm-domain-tblout-path': (
+            ['-F', '--hmm-domain-tblout-path'],
+            {'metavar': 'HMM OUTPUT FORMAT',
+             'help': "HMMER has a few output formats for including --tblout and --domtblout"
+                     "depending on the alphabet of your alignment. By default anvi'o will provide"
+                     "--tblout but if you are doing a protein search you can change to --domtblout."}
```

Comment:
I think this help message would be more clear if you instead described that users can provide a path to this argument, and the domain table output will be saved to that path. :) 
Right now it is not obvious that this will happen, and this param just sounds like a flag to turn `--domtblout` on.

Reply:
Good call. I hope this clears that up: [a9fc2f2](https://github.com/merenlab/anvio/pull/1678/commits/a9fc2f226f798588cef86ffb884ba0fff688515f)